cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help

```diff
@@ -1,47 +1,31 @@
 NAME
-       Ballerina push - Upload modules to Ballerina Central
+       ballerina push - Push module to Ballerina Central
 
 SYNOPSIS
        ballerina push <module-name> [--no-build] [--repository home]
 
 
 DESCRIPTION
-       Uploads the given module to Ballerina Central
+       Push the given module to Ballerina Central
        (https://central.ballerina.io/) repository.
 
        By default, the sources will be built before pushing
-       the module to Ballerina Central
+       the module to Ballerina Central.
 
-       Before you can push, you must enter your Ballerina Central
-       access token in Settings.toml in your home repository
-       (<USER_HOME>/.ballerina/). To get your token, register
-       on Ballerina Central and visit the user dashboard at
-       https://central.ballerina.io/dashboard.
-
-       When you push a module to Ballerina Central, the runtime will
-       validate organizations for the user against the org-name defined in
-       your project’s Ballerina.toml file. Therefore, when you have more
-       than one organization in Ballerina Central, be sure to pick the
-       organization name that you intend to push the module into and
-       set that as org-name in the project Ballerina.toml file.
+       Before pushing modules to central,
 
+       1) Ballerina central access token needs to be added to Settings.xml 
+       in home repository(<USER_HOME>/.ballerina/) 
+       before pushing modules to central. A token can be created 
```

Comment:
We don't need the "before pushing modules to central" right? Since we are starting with it?